website/blog/2021-04-12-funding-update.md

```diff
@@ -0,0 +1,57 @@
+
+<!-- truncate -->
+
+## Looking back
+
+After a few months being paid to work on Babel, Henry started receiving $11,000 USD per month. The goal was to establish the number as a baseline salary for working full-time on an open source project that seemed reasonable given the donations at the time.
+
+When we added three new paid maintainers, we couldn't afford to pay everyone the full $11,000 USD rate: Jùnliàng, Nicolò and Kai started at $2,000 USD per month, with the idea of increasing that amount when we would have received more donations.
+
+It has been more than one year since then and in the meanwhile Kai stepped down, but we still didn't have a chance to reconsider how much everyone is being paid.
+
+We have a hard time justifying the pay differences, and there isn't an obvious reason anymore for which someone should be paid more than the other team members:
+
+- We want to focus on ourselves as _people_, not as employees of a hypothetical company. We don't want to track meticulously how many hours we work, or how productive we are.
+- It's not obvious what we would consider as "working time". Part of what we do is to collaborate with other projects, but what exactly is the line between something done for Babel and something done as a personal contribution to the JavaScript ecosystem?
+- It's not possible to exactly measure how much we "produce": not every commit takes the same effort, not every task has the same impact, not every experiment has a direct result.
+
+For these reasons, we'll now change the current situation.
+
+## The future
+
+Starting from April, Henry, Nicolò and Jùnliàng will all be paid the same amount.
+
+This means we had to make a choice: does "pay everyone the same" mean that we should redistribute the total amount we spend monthly equally between us, or that we should bring everyone up to the level of who was being paid the most?
+
+We strongly believe that working in open source should be a viable and sustainable career path, so ethically we should bring everyone *up* to the same level and not *down* to the same level. However, we also need to face the fact that we don't have unlimited money: while we would love to bring everyone up to the upper level, this would mean draining [our current balance](https://opencollective.com/babel#category-BUDGET) in just a few months.
+
+We decided to take an intermediate strategy: starting from April Nicolò, Henry, and Jùnliàng will be paid $6,000 per month. This is still more than the donations we currently receive each month, but our current balance allows us to sustain this amount until the end of 2021. This gives us time to work towards attracting new big sponsors, and we'll constantly evaluate our decision to eventually adjust that amount during the year.
+
+## How you can help
+
+You can financially contribute to Babel on our Open Collective!
```

Comment:
We might also mention that we appreciate companies working with us to implement new features, like what Bloomberg did with class fields and methods.

Reply:
yeah we had some of that before in https://babeljs.io/blog/2019/11/08/babels-funding-plans#how-companies-and-individuals-have-helped-before

could mention our previous sponsors I guess, not sure